### PR TITLE
Global footer updates

### PR DESCRIPTION
**Pull request by mandythomas**
Edited _footer.html to sync with markup and link changes made in production Cascade for ADA but were lost in another recent update. Footer headings now H2 not H3 for ADA req. Footer links match live site. Social icon links contain text (though hidden with css) to pass ADA req for not having empty links.

### CODE REVIEW

**Comment by mandythomas** [comment]
also added back copyrightYear class to year in copyright line so javascript can properly run to calc the current year.

File: app/views/widgets/shared/_footer.html [hunk 1]
```diff
@@ -1,60 +1,61 @@
...
-            <li><a class="icon icon-pinterest" href="http://pinterest.com/chapmanu/" target="_blank"></a></li>
-            <li><a class="icon icon-graduation" href="http://itunes.apple.com/us/institution/chapman-university/id430678922" target="_blank"></a></li>
-            <li><a class="icon icon-youtube" href="http://www.youtube.com/user/ChapmanUniversity" target="_blank"></a></li>
-            <li><a class="icon icon-linkedin2" href="http://www.linkedin.com/company/10701?trk=tyah" target="_blank"></a></li>
-            <li><a class="icon icon-google-plus" href="https://plus.google.com/112412382370481463024/posts" target="_blank"></a></li>
-            <li><a class="icon icon-instagram" href="http://www.instagram.com/chapmanu" target="_blank"></a></li>
+            <li><a class="icon icon-twitter" href="http://twitter.com/#!/chapmanu" target="_blank"><span class="sr-only">Twitter</span></a></li>
+            <li><a class="icon icon-facebook" href="http://www.facebook.com/ChapmanUniversity" target="_blank"><span class="sr-only"Facebook">Facebook</span></a></li>
```
Comment (kaylaziegler) [inline app/views/widgets/shared/_footer.html]: Is the opening tag supposed to have this "Facebook" in it? The other ones are just `<span class="sr-only">`

Comment (mandythomas) [inline app/views/widgets/shared/_footer.html]: all the social icons should now have some text in the link but be surrounded by the span with sr-only class in order to hide it. I'm looking at the _footer.html that's part of the pull request and it looks right to me but I want to make sure I'm not misunderstanding something and there is actually an issue. The line should be the following, and I think it is. Can you look again and see if you still think it's wrong or I'm misunderstanding what you're reporting. thx:
<li><a class="icon icon-facebook" href="http://www.facebook.com/ChapmanUniversity" target="_blank"><span class="sr-only"Facebook">Facebook</span></a></li>

Comment (kaylaziegler) [inline app/views/widgets/shared/_footer.html]: @mandythomas I'm talking about this opening tag: `<span class="sr-only"Facebook">`
That third quotation mark looks incorrect to me, and inconsistent with the other tags

Comment (mandythomas) [inline app/views/widgets/shared/_footer.html]: good catch. that is a typo. It didn't break the social icons on the page so I didn't even notice. Thanks so much. I will fix

Comment (mandythomas) [inline app/views/widgets/shared/_footer.html]: @kaylaziegler I made the fix to the FB markup. So glad you caught that. Can you review and if ok, approve so I have 2 approvals before I merge to Development branch. thx

Comment (tatwell) [inline app/views/widgets/shared/_footer.html]: Ideally. But if I have at least 1 solid review and I feel it's been well vetted, I'll usually move forward.

**Pull request merged**